packages/story-editor/src/app/media/local/reducer.js

```diff
@@ -39,6 +41,7 @@ const INITIAL_STATE = {
  * @typedef {import('./typedefs').LocalMediaReducerState} LocalMediaReducerState
  */
 
+/* eslint-disable complexity */
```

Comment:
On line 17 above the complexity threshold is already set to 30.

Instead of just disabling the rule entirely here we should increase that threshold there.

Also, let's create a ticket to refactor this file here.

Reply:
fixed.

Reply:
Created a follow up ticket for this - https://github.com/google/web-stories-wp/issues/9646